From: Hilko Bengen <bengen@debian.org>
Date: Wed, 11 Apr 2018 00:42:27 +0200
Subject: Disable v2v test harness.

It needs libvirt-ocaml later than 0.6.1.4 ... which is not present in
Debian yet.
---
 v2v/test-harness/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/v2v/test-harness/Makefile.am b/v2v/test-harness/Makefile.am
index d691889..fe7c40a 100644
--- a/v2v/test-harness/Makefile.am
+++ b/v2v/test-harness/Makefile.am
@@ -30,6 +30,7 @@ SOURCES_MLI = \
 SOURCES_ML = \
 	v2v_test_harness.ml
 
+if FALSE
 if HAVE_OCAML
 if HAVE_OCAML_PKG_LIBVIRT
 
@@ -132,5 +133,6 @@ stamp-virt-v2v-test-harness.pod: virt-v2v-test-harness.pod
 
 endif
 endif
+endif
 
 .PHONY: docs
